core/src/main/java/org/apache/accumulo/core/metadata/StoredTabletFile.java

```diff
@@ -252,7 +252,7 @@ private static byte[] encodeRow(final Key key) {
     return new byte[0];
   }
 
-  private static Text decodeRow(byte[] serialized) {
+  protected static Text decodeRow(byte[] serialized) {
```

Comment:
Same comment here about removing protected

Reply:
Fixed in 3e5332d